src/Akka.Streams.Kafka/Stages/Consumers/Actors/KafkaConsumerActor.cs

```diff
@@ -56,13 +58,15 @@ internal class KafkaConsumerActor<K, V> : ActorBase, ILogReceive, IWithTimers
         /// <summary>
         /// Stores all incoming requests from consuming kafka stages
         /// </summary>
-        private IImmutableDictionary<IActorRef, KafkaConsumerActorMetadata.Internal.RequestMessages> _requests 
+        private IImmutableDictionary<IActorRef, KafkaConsumerActorMetadata.Internal.RequestMessages> _requests
             = ImmutableDictionary<IActorRef, KafkaConsumerActorMetadata.Internal.RequestMessages>.Empty;
-        
+
         /// <summary>
         /// Stores stage actors, requesting for more messages
         /// </summary>
-        private IImmutableSet<IActorRef> _requestors = ImmutableHashSet<IActorRef>.Empty;
```

Comment:
We upgraded from a hashset to a dictionary, which correlates the requested partitions to the requestor.

Reply:
So this is basically the reverse map of `_requests` now?